config/elasticsearch_client.yml

```diff
@@ -16,3 +16,7 @@ test:
 production:
   <<: *DEFAULT
   log_level: WARN
+  transport_options:
+    ssl:
+      verify: false
+  ca_fingerprint: <%= Rails.application.secrets[:ca_fingerprint] %>
```

Comment:
Will the app choke if this is here and the cookbooks haven't yet populated that key or value in the secrets.yml file?  It doesn't exist in `main` or `production` yet.

Reply:
We should deploy this at the same time as https://github.com/GSA/usasearch-cookbooks/pull/738 in order to prevent any issues.

Reply:
OK, so we have the solution, but won't merge it for a while.  Possibly several weeks.  May want to tag this PR as DO NOT MERGE then until everything else is ready.